java-client/session/src/main/java/io/deephaven/client/impl/UpdateByBuilder.java

```diff
@@ -118,6 +118,11 @@ public UpdateByColumn.UpdateBySpec visit(EmaSpec ema) {
                     .build();
         }
 
+        @Override
+        public UpdateByColumn.UpdateBySpec visit(EmsSpec spec) {
+            return null;
+        }
```

Comment:
Better to throw unsupported here IMO and link to ticket for gRPC side. I notice the rolling impls are the same, probably best to update those w/ unsupported exceptions as well.